guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
         </mediaobject>
       </screenshot>
     </sect2>
+    
+    <sect2 id="invest-sellLots">
+      <title>Automatic Calculation of Capital Gains or Losses Using Lots</title>
+       
+      <para><application>&app;</application> can automatically calculate and
+        create the capital gain or loss transactions for security sales. Lots
+        are used to link particular sell transaction splits to the buy 
+        transaction splits so that the actual cost of the sold securities can be
+        used to calculate the gain or loss.
+      </para>
+
+      <para>If you make an error, you can delete the lot(s) and/or delete the 
+        automatically created capital gain/loss transaction(s) and retry.
+      </para>
+      
+      <para><application>&app;</application> can automatically create lots or a
+        user may create lots. <application>&app;</application> can automatically
+        link buy and sell transaction splits to lots or a user may do the 
+        linking.
+      </para>
+
+      <para>
+        <application>&app;</application> can automatically <emphasis>scrub
+        </emphasis> a security account. <application>&app;</application>
```

Comment:
"GnuCash can automatically..." 3 times, and the three uses seem somewhat redundant. Can you roll it up into a single paragraph?